config/webpack/loaders/coffee.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
We don't use coffee anymore, right?

Reply:
I think we can just remove coffe @mtejedorwolox

Reply:
What about this? @mtejedorwolox @hdf1986

Reply:
Are you spying our conversations?! I am talking this with @mtejedorwolox right now